src/bin/pg_dump/pg_dump.c

```diff
@@ -12389,11 +12390,16 @@ dumpFunc(Archive *fout, const FuncInfo *finfo)
 		keyword = "FUNCTION";	/* works for window functions too */
 	
 	is_tsql_mstvf = isTsqlMstvf(fout, finfo, prokind[0], proretset[0] == 't');
+	is_tsql_itvf = isTsqlItvf(fout, finfo, prokind[0], proretset[0] == 't');
```

Comment:
Just a refactoring suggestion: Defining `getTsqlTvfType()` that returns one of NONE, ITVF or MSTVF.

Reply:
Yeah makes sense, replaced it with new function `getTsqlTvfType()`